helm/adelphi/templates/cassandra-clusters.yaml

```diff
@@ -77,7 +77,9 @@ spec:
           clusterName: {{ .Values.target.clusterName }}
           serverType: cassandra
           serverVersion: {{ .Values.target.version }}
+          {{- if .Values.target.commit_hash }}
```

Comment:
This is for #39

Reply:
How does this work exactly?  If a commit hash _isn't_ specified... how do we tell what value is used for "serverImage" here?  Is it just empty?

Reply:
@absurdfarce take a look at https://docs.datastax.com/en/cass-operator/doc/cass-operator/cassOperatorProvisioning.html#Specifytheservertypeandversion.

If the user specifies a commit_hash in Adelphi, it means he wants to use an image built from the source (say, from trunk or a dev branch), in that case Adelphi will build the image and set `serverImage` accordingly. 
Otherwise, they'll just set the `version` to some C* release version and the operator will know what to do.

It sounds like you're referring to a corner case where they specify a git hash and by chance that matches an existing C* release. That's indeed not covered here, it doesn't seem like that would happen to often though, we should advise them to use a version number instead.